proposed/vault-config-file/vault-config-file.md

```diff
@@ -0,0 +1,87 @@
+- Feature Name: ImmutableData naming base on type
+- Type: New Feature
+- Related Components: [safe_vault](https://github.com/maidsafe/safe_vault), [launcher](https://github.com/maidsafe/safe_launcher)
+- Start Date: 10-11-2015
+- RFC PR:
+- Issue number: Proposed -
+
+# Summary
+
+This RFC outlines the config file that being required by the vault to initialise personas.
+
+# Motivation
+
+## Rationale
+
+During the disscussion [safecoin implementation](https://github.com/maidsafe/rfcs/issues/61) of [RFC SafeCoin Implementation](https://github.com/maidsafe/rfcs/blob/master/proposed/0012-safecoin-implementation/0012-safecoin-implementation.md), it is being assumed that a pmid_node persona needs to be aware of it's owner's wallet address.
+
+In addition to this, when chunk_store being intialized (currently being used by PmidNode and StructruedDataManager personas only), a configurable max usable disk space needs to be specified (currently a fixed value of 1GB is being assumed to be used as default). 
```

Comment:
Should this 1Gb instead be 80% of free space (continual 80% of free space) as this will allow more default installs that perhaps do not need tweaked, but provide a nice enough default performance for the network.